examples/gatsby/README.md

```diff
@@ -0,0 +1,8 @@
+# @artsy/fresnel
+
+### Gatsby Example
+
+```
+yarn install
+yarn start
+```
```

Comment:
Love it! ✨

Reply:
@damassi There aren't currently any specific docs in the root README.md about the Gatsby example. Are you planning on adding any there?

Reply:
Oh good idea. That could be a solid selling point.